dashboards/alert_service_dashboard.json

```
@@ -1,1517 +1,1647 @@
-        "thresholds": [
-          {
-            "colorMode": "critical",
-            "fill": true,
-            "line": true,
-            "op": "gt",
-            "value": 10
-          }
-        ],
-        "timeRegions": [],
-        "title": "Kafka MQTT Divergence",
-        "tooltip": {
-          "shared": true,
-          "sort": 0,
-          "value_type": "individual"
+      "transparent": true,
+      "type": "graph",
+      "xaxis": {
+        "mode": "time",
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "format": "short",
+          "logBase": 10,
+          "show": true
         },
-        "transparent": true,
-        "type": "graph",
-        "xaxis": {
-          "mode": "time",
-          "show": true,
-          "values": []
+        {
+          "format": "short",
+          "logBase": 1,
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false
+      }
+    },
+    {
+      "aliasColors": {},
+      "bars": true,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": {
+        "type": "prometheus",
+        "uid": "PBFA97CFB590B2093"
+      },
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 12,
+        "w": 12,
+        "x": 0,
+        "y": 37
+      },
+      "hiddenSeries": false,
+      "id": 21,
+      "legend": {
+        "alignAsTable": true,
+        "avg": true,
+        "current": true,
+        "max": true,
+        "min": true,
+        "show": true,
+        "sort": "current",
+        "sortDesc": true,
+        "total": false,
+        "values": true
+      },
+      "lines": false,
+      "linewidth": 1,
+      "links": [],
+      "nullPointMode": "null",
+      "options": {
+        "alertThreshold": true
+      },
+      "percentage": false,
+      "pluginVersion": "9.3.1",
+      "pointradius": 5,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": true,
+      "steppedLine": false,
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
+          },
+          "editorMode": "code",
+          "expr": "rate(alert_service_cleared_since_startup[1m])",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "Cleared",
+          "range": true,
+          "refId": "C"
         },
-        "yaxes": [
-          {
-            "format": "short",
-            "logBase": 10,
-            "show": true
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PBFA97CFB590B2093"
           },
-          {
-            "format": "short",
-            "logBase": 1,
-            "show": true
-          }
-        ],
-        "yaxis": {
-          "align": false
+          "editorMode": "code",
+          "expr": "rate(alert_service_active_alert_since_startup[1m])",
```

Maybe it is a bit late for such comments, but in terms of metric naming conventions, here are recommendations:
https://prometheus.io/docs/practices/naming/

Following it, the name should be: `alert_service_active_alert_total`

---

In case of the query itself, I would suggest two improvements:

1. `irate` over 5m instead of `rate` over 1m
2. adding sum to capture restarts and multiple instances

```suggestion
          "expr": "sum(irate(alert_service_active_alert_since_startup[5m]))",
```

---

The same comment goes to other panels as well.